Seasons Defense/Assets/Scripts/TowerController.cs

```diff
@@ -4,24 +4,21 @@ public class TowerController : MonoBehaviour
 {
     public GameObject missilePrefab;
     public float offset;
+    
+    public void Fire() {
 
-    private void Update()
-    {
-        if (Input.GetMouseButtonDown(1))
-        {
-            Vector3 mousePosition = new Vector3(Input.mousePosition.x, Input.mousePosition.y, Camera.main.nearClipPlane);
```

Comment:
Not sure how nearClipPlane is used here, but I may be unfamiliar with it.

Can we make a private or hidden public reference to the Camera.main at Start() to avoid the expensive calls on this line and below?

Reply:
Actually we can make the z-axis just a zero. Thus I think that's better for the time being